[2024-11-02T18:57:20.068+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-02T18:57:20.085+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_data.get_db_connection_params manual__2024-11-02T18:57:18.167244+00:00 [queued]>
[2024-11-02T18:57:20.093+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_data.get_db_connection_params manual__2024-11-02T18:57:18.167244+00:00 [queued]>
[2024-11-02T18:57:20.094+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-02T18:57:20.108+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_db_connection_params> on 2024-11-02 18:57:18.167244+00:00
[2024-11-02T18:57:20.114+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=136) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-02T18:57:20.115+0000] {standard_task_runner.py:72} INFO - Started process 138 to run task
[2024-11-02T18:57:20.116+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'transform_data', 'get_db_connection_params', 'manual__2024-11-02T18:57:18.167244+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/transform_data.py', '--cfg-path', '/tmp/tmpyuzcrns6']
[2024-11-02T18:57:20.118+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask get_db_connection_params
[2024-11-02T18:57:20.166+0000] {task_command.py:467} INFO - Running <TaskInstance: transform_data.get_db_connection_params manual__2024-11-02T18:57:18.167244+00:00 [running]> on host b08ab6761509
[2024-11-02T18:57:20.243+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transform_data' AIRFLOW_CTX_TASK_ID='get_db_connection_params' AIRFLOW_CTX_EXECUTION_DATE='2024-11-02T18:57:18.167244+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-02T18:57:18.167244+00:00'
[2024-11-02T18:57:20.244+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-02T18:57:20.247+0000] {python.py:240} INFO - Done. Returned value was: {'url': 'postgresql+psycopg2://postgres@host.docker.internal:5432/metrodata'}
[2024-11-02T18:57:20.268+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-02T18:57:20.269+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=transform_data, task_id=get_db_connection_params, run_id=manual__2024-11-02T18:57:18.167244+00:00, execution_date=20241102T185718, start_date=20241102T185720, end_date=20241102T185720
[2024-11-02T18:57:20.297+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-02T18:57:20.322+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-02T18:57:20.323+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
